common/lib/xmodule/xmodule/modulestore/mongo/draft.py

```diff
@@ -624,6 +624,9 @@ def has_changes(self, xblock):
             return True
         # if this block doesn't have changes, then check its children
         elif xblock.has_children:
+            # fix a bug where dangling pointers should imply a change
+            if len(xblock.children) > len(xblock.get_children()):
```

Comment:
Yes, get_children iterates through children and gets the xblocks. If any ItemNotFound occur, it ignores those.